cypress/e2e/spec/modifyFilter.feature

```diff
@@ -0,0 +1,24 @@
+Feature: 'Modify  Filter'
+
+  Background: Create filter
+    Given user clicks on side bar filter button
+    Then user clicks on 'ADD_FILTER' button
+    And user verifies demo data 'Demo Api Tests' is visible
+  @modify
+  Scenario Outline: 'As a Use I want to be able to modify filter by setting <bugType> bug type'
+    Then user clicks on more condition button
+    And user selects <bugType> condition
+    And user scrolls into view of 'TOOLBAR'
+    Then user verifies filter label 'FIELD_FILTER' has a value <bugType>
+    And user sets value 'Demo Api' into 'Enter name' input field
+    When user sets value <filterNumber> into 'Enter quantity' input field
+    Then user see <filteredLaunches> number of filtered launches
+    And user clicks on 'SAVE' button
+    And user sets value 'New filter for bug type`' into 'Enter filter name' input field
+    And user adds new filter
+    Then user verifies 'FILTER_NAME' link contains 'New filter for bug type' text
+    Examples:
+      | bugType                | filterNumber | filteredLaunches |
+      | 'Product Bug Group'    | '2'          | 50               |
+      | 'Automation Bug Group' | '4'          | 2                |
+      | 'Failed'               | '5'          | 4                |
```

Comment:
As an exercise, I asked chapGPT to rewrite/review this TC using good practices and enhance it for better clarity and readability. Here is what it suggested me:

```
Feature: Modify Filter

  Background: 
    Given I open the filter panel from the sidebar
    And I add a new filter by clicking 'ADD_FILTER'
    And I confirm that 'Demo Api Tests' data is visible

  @modify
  Scenario Outline: Modify an existing filter by setting a specific bug type
    Given I am editing a filter
    When I add a condition for bug type "<bugType>"
    And I navigate to the 'TOOLBAR'
    Then the 'FIELD_FILTER' should display the bug type "<bugType>"
    When I enter 'Demo Api' as the filter name
    And I specify the quantity "<filterNumber>" for the filter
    Then the filter should show "<filteredLaunches>" launches
    When I save the filter with the name 'New filter for bug type'
    Then the filter list should include a filter named 'New filter for bug type'

    Examples:
      | bugType                | filterNumber | filteredLaunches |
      | Product Bug Group      | 2            | 50               |
      | Automation Bug Group   | 4            | 2                |
      | Failed                 | 5            | 4                |
```

```
Changes Made:
1. Language and Context: Improved clarity and context in steps (e.g., specifying actions like "navigate" and "display" for a better understanding of the GUI interactions).
2. Removal of Quotes: Removed unnecessary quotes around parameters and values for simplicity.
3. Clarification of Actions and Expectations: More precise language to delineate actions from expectations, making it clear what the user does and what the system should do as a response.
4. Scenario Title: More descriptive and user-centric title.
5. Consistency and Readability: Consistent verb tenses and simplified steps to enhance readability and comprehension.
```
What do you think? Makes sense?